mlflow/utils/mime_type_utils.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
I made a new utils module for this but if you think it belongs in an existing module (like rest_utils or maybe file_utils) I'm happy to move it.

Reply:
This is fine.